lib/online_order.rb

```diff
@@ -0,0 +1,122 @@
+require 'csv'
+require 'awesome_print'
+require_relative 'order.rb'
+
+
+#_______________ Grocery::OnlineOrder _______________
+
+module Grocery
+  class OnlineOrder < Grocery::Order
+
+    attr_reader :id, :products, :customer_id, :status
```

Comment:
Since `OnlineOrder` inherits from `Order` you don't need to create readers for `id` or `products`.  Instead just use the ones created in `Order`.  

This should be:
```ruby
attr_reader :customer_id, :status
```